.github/workflows/build.yaml

```diff
@@ -1,14 +1,14 @@
 name: Build Node.js backend and deploy to AWS ECR
-on:   
+on:
   push:
-    branches: 
+    branches:
       - 'CB**'
       - prod
       - uat
       - main
 
   pull_request:
-    branches: 
+    branches:
```

Comment:
from ur pr is deleteing all the white space, and meanwile ryan   is adding all the white spaces